streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -141,7 +144,11 @@ public void openDB(ProcessorContext context) {
                 valueSerde == null ? (Serde<V>) context.valueSerde() : valueSerde);
 
         this.dbDir = new File(new File(context.stateDir(), parentDir), this.name);
-        this.db = openDB(this.dbDir, this.options, TTL_SECONDS);
+        try {
+            this.db = openDB(this.dbDir, this.options, TTL_SECONDS);
```

Comment:
Just for my own benefits: what's the advantage of using `Files` here, and in line 173 below?